man/mono.1

```diff
@@ -65,6 +65,45 @@ and
 See the description for 
 .B MONO_IOMAP
 in the environment variables section for more details.
+.SH METHOD DESCRIPTIONS
+A number of diagnostic command line options take as argument a method
+description.   A method description is a textual representation that
```

Comment:
Modern form is one space after period, but I admit I've never written a man page, perhaps they have different conventions, or there was a desire to stretch toward equal length lines?